tests/test_audio.py

```diff
@@ -28,14 +28,15 @@
 
 import telegram
 from tests.base import BaseTest, timeout
+from tests.reupload import get_file_id
 
 
 class AudioTest(BaseTest, unittest.TestCase):
     """This object represents Tests for Telegram Audio."""
 
     def setUp(self):
         self.audio_file = open('tests/data/telegram.mp3', 'rb')
-        self.audio_file_id = 'CQADAQADDwADHyP1B6PSPq2HjX8kAg'
+        self.audio_file_id = get_file_id('audio')
```

Comment:
Actually.... are you sure? In get_file_id there's a test to see if it's already been uploaded... wouldn't that be global across all tests in theory?